package.json

```diff
@@ -50,7 +50,9 @@
     "node-mocks-http": "1.2.3",
     "sinon": "1.17.6",
     "typescript": "2.0.10",
-    "typings": "2.0.0"
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
Hmmm, good point. Node < 4.x isn't supported anymore, but customers might still be using it. @beckylino can we check what versions of Node an ikey is reporting from? I couldn't find that in our metrics.

In any case makes sense to bump the @types/node version back to 4.x, done.